pkg/container/container.go

```diff
@@ -115,6 +126,17 @@ func (c Container) ImageName() string {
 	return imageName
 }
 
+// NewImageName returns the name of the Docker image that should be used to
```

Comment:
Maybe CurrentImageName is more clear.

Reply:
it's an image to upgrade to, so current doesn't explain that. it could be UpgradeImageName